### PR TITLE
ENT-3719 MSI package module

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @craigcomstock can review this?

File: reference/promise-types/packages.markdown [hunk 1]
```diff
@@ -526,3 +526,34 @@ Manage packages using Slackware's [slackpkg](https://slackpkg.org).
 **History:**
 
 * Added in CFEngine 3.12.0
+
+### msiexec
+
+Manage MSI packages using MSI installer on Windows.
+
+Due to lack of central software repository on supported versions of Windows,
+neither installation from repository nor checking for upgrades is not supported.
```
Comment (nickanderson) [inline reference/promise-types/packages.markdown]: Double negative.

d/not/

File: reference/promise-types/packages.markdown [hunk 2]
```diff
@@ -526,3 +526,34 @@ Manage packages using Slackware's [slackpkg](https://slackpkg.org).
...
+You need to provide full path to MSI pakage file to install, or installed
```
Comment (nickanderson) [inline reference/promise-types/packages.markdown]: The full path to the MSI package file must be supplied in order to promise the package is installed. In order to promise a package is absent (not installed) the package name must be used.

File: reference/promise-types/packages.markdown [hunk 3]
```diff
@@ -526,3 +526,34 @@ Manage packages using Slackware's [slackpkg](https://slackpkg.org).
...
+        package_module => msiexec;
+
+      "Google Update Helper"
+        policy => "absent",
+        package_module => msiexec;
+```
+
+**History:**
+
+* Added in CFEngine 3.14.0
```
Comment (nickanderson) [inline reference/promise-types/packages.markdown]: We are backporting this to 3.12.x aren't we? This can be updated when we do it.

Comment (Lex-2008) [reply reference/promise-types/packages.markdown]: Right. But we can already add this info

**Pull request merged**